qiskit_ibm_runtime/options/twirling_options.py

```diff
@@ -36,12 +36,23 @@ class TwirlingOptions:
          those measurement registers not involved within a conditional logic.
          Default: True for Estimator, false for Sampler.
 
-        num_randomizations: The number of random samples to use when twirling or
-            peforming sampled mitigation. If "auto", the value will be chosen automatically
-            based on the input PUBs. Default: "auto".
+        num_randomizations: The number of random samples to use when twirling or peforming sampled mitigation.
+          If `num_randomizations` is "auto":
```

Comment:
Here and elsewhere, recall that in sphinx/ReST, single backticks are for italics, and double backticks are for inline code segments.

```python
          If ``num_randomizations`` is ``"auto"``:
```